Packs/CortexAttackSurfaceManagement/README.md

```diff
@@ -20,128 +20,162 @@ Automated remediation is only possible when the right conditions are met.  These
+
+![Cortex ASM - AWS Enrichment](https://raw.githubusercontent.com/demisto/content/master/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_AWS_Enrichment.png)
+
+#### Cortex ASM - Azure Enrichment
+
+Playbook that given the IP address enriches Azure information relevant to ASM alerts.
+
+![Cortex ASM - Azure Enrichment](https://raw.githubusercontent.com/demisto/content/master/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Azure_Enrichment.png)
+
+#### Cortex ASM - Decision
+
+Playbook that returns "RemediationAction" options based on meeting "Automated Remediation Requirements" as well as whether ServiceNowV2 integration is set up.
+
+![Cortex ASM - Decision](https://raw.githubusercontent.com/demisto/content/master/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Decision.png)
+
 #### Cortex ASM - Detect Service
 
 Playbook that looks at what ASM sub-type the alert is and directs it to different pre/post mitigation scans (such as NMAP, SNMP).
+
 ![Cortex ASM - Detect Service](https://raw.githubusercontent.com/demisto/content/master/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Detect_Service.png)
 
 #### Cortex ASM - Enrichment
 
 Playbook that is used as a container folder for all enrichments of ASM alerts.
+
 ![Cortex ASM - Enrichment](https://raw.githubusercontent.com/demisto/content/master/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Enrichment.png)
 
-#### Cortex ASM - AWS Enrichment
+#### Cortex ASM - GCP Enrichment
 
-Playbook that given the IP address enriches AWS information relevant to ASM alerts.
-![Cortex ASM - AWS Enrichment](https://raw.githubusercontent.com/demisto/content/master/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_AWS_Enrichment.png)
+Playbook that given the IP address enriches GCP information relevant to ASM alerts.
 
-#### Cortex ASM - ServiceNow CMDB Enrichment
+![Cortex ASM - GCP Enrichment](https://raw.githubusercontent.com/demisto/content/master/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_GCP_Enrichment.png)
 
-Playbook that given the IP address enriches ServiceNow CMDB information relevant to ASM alerts.
-![Cortex ASM - ServiceNow CMDB Enrichment](https://raw.githubusercontent.com/demisto/content/master/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_ServiceNow_CMDB_Enrichment.png)
+#### Cortex ASM - Rapid7 Enrichment
 
-#### Cortex ASM - Tenable.io Enrichment
+Playbook that given the IP address enriches Rapid7 information relevant to ASM alerts.
 
-Playbook that given the IP address enriches Tenable.io information relevant to ASM alerts.
-![Cortex ASM - Tenable.io Enrichment](https://raw.githubusercontent.com/demisto/content/master/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Tenable.io_Enrichment.png)
+![Cortex ASM - Rapid7 Enrichment](https://raw.githubusercontent.com/demisto/content/master/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Rapid7_Enrichment.png)
```

Comment:
You can change this later (separate PR right away or when the playbook is edited in the future.

```suggestion
![Cortex ASM - Rapid7 Enrichment](https://raw.githubusercontent.com/demisto/content/8f2a866b666627cb0c6c7ea860e7f1337b4766b7/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Rapid7_Enrichment.png)
```

Reply:
Resolved in 70f2b98.